practice_app/pages/api/getStatistics.js

```diff
@@ -0,0 +1,65 @@
+import axios from 'axios'
+
+const { checking } = require('./getStatisticsModule')
+
+export default (req, res) => {
+    // req object contains whatever information we can get from the request that comes from the client (from the browser)
+    // res object is used tow write our reponse
+    const { query, method } = req
+
+    switch (method) {
+        case 'GET':
+            {
+                // ...
+                res.statusCode = 200
+
```

Comment:
You are setting the status code twice, once here, and once at the end. Remove this